File: packages/tools/check_wechat.js
import { execSync } from 'node:child_process';
try {
    const result = execSync('tasklist', { encoding: 'utf8' });
    const lines = result.split('\n').filter(l => l.toLowerCase().includes('wechat'));
    if (lines.length > 0) {
        console.log('✅ 微信正在运行:');
        lines.forEach(l => console.log(l));
    } else {
        console.log('❌ 微信未运行');
    }
} catch (e) {
    console.log('Error:', e.message);
}

File: packages/tools/check_wechat.py
import subprocess
import sys
import os

print("CWD:", os.getcwd())

try:
    result = subprocess.run(['tasklist'], capture_output=True, text=True)
    output = result.stdout
    wechat_lines = [l for l in output.split('\n') if 'WeChat' in l]
    if wechat_lines:
        print("[OK] WeChat is running:")
        for line in wechat_lines:
            print(line)
    else:
        print("[NOT FOUND] WeChat not running")
except Exception as e:
    print(f"Error: {e}")
    sys.exit(1)
